src/opera/pge/dist_s1/dist_s1_pge.py

```diff
@@ -37,6 +39,143 @@ class DistS1PreProcessorMixin(PreProcessorMixin):
     _pre_mixin_name = "DistS1PreProcessorMixin"
     _valid_input_extensions = (".tif",)
 
+    def _validate_rtcs(self):
```

Comment:
I love all the validations you've implemented here, but I also feel like the method itself is a little unwieldy in terms of length and the number of sub-functions within. 

Could we break out the validation steps you've numbered in the docstring into separate methods in the pre-processor? This `_validate_rtcs` method could then be used to roundup the calls to each of the individual validation methods such that we still only need to call call `self._validate_rtcs()` within `run_preprocessor()`.

Reply:
@collinss-jpl This is done. Let me know if this is what you had in mind. I did also have to copy some bits that were common to each function